users/faq.rst

```diff
@@ -524,3 +524,21 @@ separate file, i.e. you should run::
 This only takes effect after a reboot. To adjust the limit immediately, run::
 
     sudo sh -c 'echo 204800 > /proc/sys/fs/inotify/max_user_watches'
+
+How do I reset the GUI password?
+--------------------------------
+
+If you've forgotten/lost the GUI password, you can reset it to blank by deleting the **<user>** and **<password>** XML tags from the **<gui>** block in config.xml. The location of the file depends on OS and is described in the configuration documentation.
+
+For example the two lines in bold below would be removed from the file.
+
+| <gui enabled="true" tls="false" debugging="false">
+|    <address>127.0.0.1:8384</address>
+|    **<user>syncguy</user>**
+|    **<password>$2a$10$s9wWHOQetp46Cq7GPye69.KqKantdUrbHCeLnZ9A6VlzkEPPYHDAC</password>**
+|    <apikey>9RCKohqCAyrj5RjpyZdR2wXmQ9PyQFeN</apikey>
+|    <theme>default</theme>
+| </gui>
+|
+
+After this the GUI can be accessed without authentication. No restart of Syncthing or refresh of the browser is *required*, but for consistent results it is recommended.
```

Comment:
I think this is not true, as it would only work if the config is reloaded. Also, syncthing might decide to save the config overwriting changes you've made manually, so this operation should be done while syncthing is not running.

Reply:
I was surprised myself to see the change take effect without restart, it's possible I misled myself into only thinking that's what I did. It is probably wise to stop Syncthing as standard procedure anyway, whether necessary or not. I'll run through the procedure a couple times and rewrite.